app/models/choice.rb

```diff
@@ -0,0 +1,7 @@
+class Choice < ApplicationRecord
+    belongs_to :word, foreign_key: "word_id"
+
+    validates :choices, presence: true, length: { minimum: 1 }
+    validates :correct_ans, inclusion: { in: [true, false] }
```

Comment:
Remove this one because you don't need it; and it will cause unnecessaery validations in the future.
![Screen Shot 2021-05-05 at 9 43 53 PM](https://user-images.githubusercontent.com/35232348/117150803-1075af80-adeb-11eb-8d6f-2f7cd31cc0a9.png)